### PR TITLE
Improve DiracFile removals

### PR DESCRIPTION
**Pull request by mesmith75**
Dirac's `removeFile` can take a list of LFNs so it is quicker to do that if we are removing multiple files. Therefore I have added the massRemove function to DiracFile.

Once this and #1445 have been merged I'll put the massRemove into the auto output remove in the job to be as quick as possible.

### CODE REVIEW

File: ganga/GangaDirac/Lib/Files/DiracFile.py [hunk 1]
```diff
@@ -403,6 +403,49 @@ def remove(self):
...
+                logger.warning(msg)
+                return False
+            #Check that the number of removed files is the same as the number requested
+            if result.get('Successful', False):
+                _successes.extend(result['Successful'].keys())
+            if result.get('Failed', False):
+                _failures.extend(result['Failed'].keys())
+            i = i+200
+
+        def clearFileInfo(f):
```
Comment (egede) [inline ganga/GangaDirac/Lib/Files/DiracFile.py:440]: You are changing information of objects that might sit inside a completed job. Will the job get correctly marked as dirty and the change persisted?

Comment (mesmith75) [reply ganga/GangaDirac/Lib/Files/DiracFile.py:440]: I'm not sure how one would mark the job. The change of the DiracFile object should be persisted though.

Comment (egede) [reply ganga/GangaDirac/Lib/Files/DiracFile.py:440]: If it works, I am not too worried.

**Review by egede — APPROVED** [review]

**Comment by mesmith75** [comment]
I don't think I like this. I will try to do it better.

**Pull request closed without merging**